hbw/app/assets/stylesheets/hbw/brand_variables.sass

```diff
@@ -45,13 +45,12 @@
   // a +claiming table +claiming popup
   --base-hbw-black: black
   //claiming-count +hbw-icon-svg-path + tabs -> a border
-  --base-hbw-brand-success: #{$brand-success}
+  --base-hbw-brand-success: #18BC9C
   //border + color in div
-  --base-hbw-text-color: #{$text-color}
+  --base-hbw-text-color: #2C3E50
   --base-hbw-priority-medium-color: #926902
-  --base-hbw-priority-high-color: #{$brand-warning}
+  --base-hbw-priority-high-color: #F39C12
   --base-hbw-urgent-color: #E74C3C
-  --base-hbw-task-overview-first-child-color: #4A4A4A
```

Comment:
Removed with its using in `branding.sass`, now left column in task overview has #2C3E50 color like right column